modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ViewObjectDefinitions/ModalBindToRootObject.tsx

```diff
@@ -0,0 +1,284 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayLabel from '@clayui/label';
+import ClayLoadingIndicator from '@clayui/loading-indicator';
+import ClayModal, {useModal} from '@clayui/modal';
+import React, {useEffect, useState} from 'react';
+
+import './ModalBindToRootObject.scss';
+
+import ClayButton from '@clayui/button';
+import {
+	Input,
+	SingleSelect,
+	getLocalizableLabel,
+	openToast,
+} from '@liferay/object-js-components-web';
+import {createResourceURL, fetch} from 'frontend-js-web';
+
+interface ModalBindToRootObjectProps {
+	baseResourceURL: string;
+	onvisibilityChange: () => void;
```

Comment:
@danvb112 Just a small change, please capitalize the 'v' on `onVisibilityChange`.